src/webgpu/api/validation/encoding/cmds/copyTextureToTexture.spec.ts

```diff
@@ -510,6 +511,38 @@ g.test('copy_within_same_texture')
     );
   });
 
+g.test('copy_aspects')
+  .params(
+    params()
+      .combine(poptions('format', ['rgba8unorm', 'depth24plus-stencil8'] as const))
+      .combine(poptions('sourceAspect', ['all', 'depth-only', 'stencil-only'] as const))
+      .combine(poptions('destinationAspect', ['all', 'depth-only', 'stencil-only'] as const))
```

Comment:
Can we also test that `depth-only` is valid for depth-only formats, and a TODO to test `stencil-only` is valid for `stencil8`?

Reply:
In current Dawn using ```depth-only```  for ```depth32float``` will report a validation error:
 "Texture aspect must be "all" for texture to texture copies"

I will fix it in Dawn.

Reply:
Sounds good, but you just filing a bug is ok too (so you don't need to multi-task as much).